sdk/web-pubsub/web-pubsub-client/README.md

```diff
@@ -0,0 +1,211 @@
+# Azure Web PubSub CloudEvents handlers for Express
+
+[Azure Web PubSub service](https://aka.ms/awps/doc) is an Azure-managed service that helps developers easily build web applications with real-time features and publish-subscribe pattern. Any scenario that requires real-time publish-subscribe messaging between server and clients or among clients can use Azure Web PubSub service. Traditional real-time features that often require polling from server or submitting HTTP requests can also use Azure Web PubSub service.
+
+When a WebSocket connection connects, the Web PubSub service transforms the connection lifecycle and messages into [events in CloudEvents format](https://docs.microsoft.com/azure/azure-web-pubsub/concept-service-internals#workflow). This library provides an express middleware to handle events representing the WebSocket connection's lifecycle and messages, as shown in below diagram:
+
+![cloudevents](https://user-images.githubusercontent.com/668244/140321213-6442b3b8-72ee-4c28-aec1-127f9ea8f5d9.png)
+
+Details about the terms used here are described in [Key concepts](#key-concepts) section.
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/web-pubsub/web-pubsub-express) |
```

Comment:
README talks a lot about `web-pubsub-express` but this is `web-pubsub-client`, no?

Reply:
Sorry, the README has not been fully updated. I may update it in the next PR